kolibri/plugins/video_mp4_render/assets/src/vue/index.vue

```diff
@@ -26,44 +26,44 @@
 
     props: ['files'],
 
-    methods: {
-
-      setPlayState(state) {
-        if (state === true) {
-          this.videoPlayer.$('.videotoggle').classList.add('videopaused');
-          this.videoPlayer.$('.videoreplay').classList.add('display');
-          this.videoPlayer.$('.videoforward').classList.add('display');
-        } else {
-          this.videoPlayer.$('.videotoggle').classList.remove('videopaused');
-          this.videoPlayer.$('.videoreplay').classList.remove('display');
-          this.videoPlayer.$('.videoforward').classList.remove('display');
-        }
-      },
-
-      getPosterSource() {
+    computed: {
+      posterSource() {
         return this.files.filter(
           (file) => file.extension === 'png' | file.extension === 'jpg'
         )[0].storage_url;
       },
 
-      getVideoSources() {
+      videoSources() {
         return this.files.filter(
           (file) => file.extension === 'mp4' | file.extension === 'webm' | file.extension === 'ogg'
```

Comment:
Works for now - but we may want to have a variable defined that just lists the videoFileExtensions, posterFileExtensions etc...

`const VideoFileExtensions = ['mp4', 'webm', 'ogg'];`
This would then become:
```
return this.files.filter(
    (file) => videoFileExtensions.some((ext) => ext === file.extension)
);
```